renku/service/logging.yaml

```diff
@@ -0,0 +1,31 @@
+---
+formatters:
+  simple:
+    format: "[%(asctime)s] [%(levelname)s] [%(name)s] %(message)s"
+handlers:
+  console:
+    class: logging.StreamHandler
+    formatter: simple
+    level: DEBUG
+    stream: "ext://sys.stdout"
+loggers:
+  service:
+    handlers:
+      - console
+    level: DEBUG
```

Comment:
can we make the level configurable through an environment variable so we can easily toggle it in a running service?

Reply:
Right, I have to test this a bit more after rebase, but this should be already in place and this config should be only for default value.

Reply:
are they exposed through the chart?

Reply:
I just checked and one is and one is not, I'll add them so both have explicitly set default values there. :+1: